### PR TITLE
feat(nuxt): Use Sentry wizard in nuxt onboarding

### PR DESCRIPTION
**Pull request by andreiborza**
Screenshot 2024-11-29 at 18 06 06@2x

### CODE REVIEW

**Review by priscilawebdev — APPROVED** [review]
tested everything and it looks good to me! 🚀

File: static/app/locale.tsx [hunk 1]
```diff
@@ -407,6 +407,14 @@ export function gettextComponentTemplate(
   return mark(renderTemplate(parsedTemplate, components) as JSX.Element);
 }
 
+/**
+ * Helper over `gettextComponentTemplate` with a pre-populated `<code />` component that
+ * is commonly used.
+ */
+export function tctCode(template: string, components: ComponentMap = {}) {
```
[the remark this answers is not in the document]
Comment (andreiborza) [reply static/app/locale.tsx:414]: Was considering replacing all occurrences with this but wanted to see what people thought of it first.

**Review by s1gr1d — APPROVED** [review]
Looks good, thanks 🙌

File: static/app/locale.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (priscilawebdev) [reply static/app/locale.tsx:414]: :D `tctc` sounds interesting!

**Pull request merged**